manager/src/main/java/org/openremote/manager/security/ManagerIdentityProvider.java

```diff
@@ -185,7 +185,7 @@ static User[] getUsersFromDb(PersistenceService persistenceService, UserQuery qu
                     // exists otherwise it will just match any other row
                     sb.append("NOT ");
                 }
-                sb.append("EXISTS (SELECT urm.user_id from public.user_role_mapping urm join public.keycloak_role kr on urm.role_id = kr.id where urm.user_id = u.id and not kr.client_role and");
+                sb.append("EXISTS (SELECT urm.user_id from public.user_role_mapping urm join public.keycloak_role kr on urm.role_id = kr.id where urm.user_id = u.id and");
```

Comment:
Apparently the `alarms:write` role is a client role and would previously be filtered out. Without this change it would not be possible to query users having this role which is used to determine who should be send an e-mail in case an alarm has severity HIGH and no assignee. Querying users based on roles only seems to be done by the `UserResourceTest`.

Reply:
This was to query `realmRoles`, user query doesn't seem to support client role searching so this should be added separately if needed.

I don't know why you would need `alarms:write` to recieve a notification but will look at that separately.